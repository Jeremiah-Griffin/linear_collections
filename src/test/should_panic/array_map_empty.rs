pub fn main() {
    macros::array_map![];
}
