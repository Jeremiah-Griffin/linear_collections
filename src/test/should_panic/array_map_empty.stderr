error: proc macro panicked
 --> src/test/should_panic/array_map_empty.rs:2:5
  |
2 |     macros::array_map!([]);
  |     ^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Input may not be empty
